dask/blockwise.py

```diff
@@ -381,23 +445,15 @@ def __dask_distributed_pack__(
         # All blockwise tasks will depend on the futures in `indices`
         global_dependencies = {stringify(f.key) for f in indices_unpacked_futures}
 
-        # Handle `io_deps` serialization.
-        # If `io_deps[<collection_key>]` is just a dict, we rely
-        # entirely on msgpack.  It is up to the `Blockwise` layer to
-        # ensure that all arguments are msgpack serializable. To enable
-        # more control over serialization, a `BlockwiseIODeps` mapping
-        # subclass can be defined with the necessary
-        # `__dask_distributed_{pack,unpack}__` methods.
+        # Handle `io_deps` serialization. Assume each element
+        # is a `BlockwiseDep`-based object.
         packed_io_deps = {}
         for name, input_map in self.io_deps.items():
```

Comment:
```suggestion
        for name, blockwise_dep in self.io_deps.items():
```
nit: maybe rename this variable for clarity?